### ISSUE TITLE
Update tracing module

### ISSUE THREAD

**Issue by nerox8664** [nerox8664/pytorch2keras#84]
Due to new PyTorch updates the only way to proper handle weights and layer names at the same time - to use ONNX export. 

From the next version of the converter, tracing module will be replaced. Hopefully it will open new features - converting modules such as ModuleList, converting recurrent networks and so on. Stay tuned.

The new tracing module will be placed to the separated repository [https://github.com/nerox8664/onnx2keras](https://github.com/nerox8664/onnx2keras) and used as a library after implementation.

Progress:
- [x] Model loading / parsing
- [x] Convert convolution layers:
  - [x] Conv2d
  - [x] ConvTranspose2d
- [x] Convert linear layers:
  - [x] Linear (Gemm / MatMul)
- [x] Convert pooling layers:
  - [x] MaxPool
  - [x] AveragePool
  - [x] GlobalAveragePool
- [x] Convert padding layers:
  - [x] Pad*
- [x] Convert normalization layers:
  - [x] BatchNorm2d
  - [x] InstanceNorm2d
- [x] Activations:
  - [x] ReLU
  - [x] LeakyReLU
  - [x] Sigmoid
  - [x] Tanh
  - [x] Selu
- [x] Reshape operations:
  - [x] View / Flatten *
  - [x] Transpose *
  - [x] Unsqueeze *
  - [x] Shape *
- [x] Also element-wise operations:
  - [x] Div
  - [x] Add
  - [x] Mul
  - [x] Sub
- [x] Other operations:
  - [x] Gather *
  - [x] Clip
  - [x] Exp
  - [x] Log
- [x] Implement new tests to cover converter
- [x] Test new available models on torchvision
